src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,367 @@
+
+```JSON
+{
+    "SkullStripped": true,
+    "Resolution": {
+        "1": "Matched with MNI152NLin6Asym 1.6mm isotropic",
+        "2": "Matched with MNI152NLin6Asym 2.0mm isotropic"
+    },
+    "Density": {
+        "10k": "10242 vertices per hemisphere (5th order icosahedron)",
+        "41k": "40962 vertices per hemisphere (6th order icosahedron)"
+    }
+}
+```
+
+## Masks
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_keywords>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_mask.nii.gz
+```
+
+A binary (1 - inside, 0 - outside) mask in the space defined by `<space>`. By
+default (i.e., if no transformation has taken place) the value of `space` should
+be set to `orig`.
+
+JSON metadata fields:
+
+| **Key name** | **Description**                                                                                                                                |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| RawSources   | Same as defined in [Introduction][intro], but elevated from OPTIONAL to REQUIRED                                                               |
+| Type         | RECOMMENDED. Short identifier of the mask. Reserved values: `Brain` - brain mask, `Lesion` - lesion mask, `Face` - face mask, `ROI` - ROI mask |
+| Resolution   | REQUIRED if `res` is present. String or dictionary of label to string. Specifies the interpretation of the resolution keyword.                 |
+| Density      | REQUIRED if `den` is present. String or dictionary of label to string. Specifies the interpretation of the density keyword.                    |
+
+Examples:
+
+```Text
+func_loc/
+    sub-001/
+        func/
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.nii.gz
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.json
+```
+
+```Text
+manual_masks/
+    sub-001/
+        anat/
+            sub-001_desc-tumor_mask.nii.gz
+            sub-001_desc-tumor_mask.json
+```
+
+## Segmentations and parcellations
+
+Common JSON metadata fields:
+
+| **Key name** | **Description**                                                                                                                |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Manual       | OPTIONAL. Boolean. Indicates if the segmenation was performed manually or via an automated process                             |
+| Atlas        | OPTIONAL. Which atlas (if any) was used to derive the segmentation.                                                            |
+| Resolution   | REQUIRED if `res` is present. String or dictionary of label to string. Specifies the interpretation of the resolution keyword. |
```

Comment:
Please use the formal JSON terminology of `object` instead of `dict`, which is Pythonese. See https://www.json.org/json-en.html and https://github.com/bids-standard/bids-specification/pull/265#issuecomment-634276265